### DISCUSSION TITLE
System calls?

### DISCUSSION THREAD

**Discussion by Melab** [CosmosOS/Cosmos#3022] [General]
Does this OS not have system calls? I looked for a list of system calls like I would see in ither kernels' source code, but I can't find one.

**Comment by ascpixi** [comment]
Not out of the box, no. Cosmos itself does not define support for executing binaries outside of the kernel itself, so defining system calls with no entity to use said calls wouldn't make much sense. It's up to the end-user to implement those - this is by design.

**Comment by zarlo** [comment]
we will never likely have a build in syscall system as we feel it is out of scope of the main project we might do what we did with http and ftp
